python/dgl/distributed/partition.py

```diff
@@ -96,27 +114,27 @@ def load_partition(part_config, part_id, load_feats=True):
     assert EID in graph.edata, "the partition graph should contain edge mapping to global edge ID"
 
     gpb, graph_name, ntypes, etypes = load_partition_book(part_config, part_id, graph)
-    ntypes_list, etypes_list = [], []
-    for ntype in ntypes:
-        ntype_id = ntypes[ntype]
-        # graph.ndata[NID] are global homogeneous node IDs.
-        nids = F.boolean_mask(graph.ndata[NID], _get_inner_node_mask(graph, ntype_id))
-        partids1 = gpb.nid2partid(nids)
-        _, per_type_nids = gpb.map_to_per_ntype(nids)
-        partids2 = gpb.nid2partid(per_type_nids, ntype)
-        assert np.all(F.asnumpy(partids1 == part_id)), 'load a wrong partition'
-        assert np.all(F.asnumpy(partids2 == part_id)), 'load a wrong partition'
-        ntypes_list.append(ntype)
-    for etype in etypes:
-        etype_id = etypes[etype]
-        # graph.edata[EID] are global homogeneous edge IDs.
-        eids = F.boolean_mask(graph.edata[EID], _get_inner_edge_mask(graph, etype_id))
-        partids1 = gpb.eid2partid(eids)
-        _, per_type_eids = gpb.map_to_per_etype(eids)
-        partids2 = gpb.eid2partid(per_type_eids, etype)
-        assert np.all(F.asnumpy(partids1 == part_id)), 'load a wrong partition'
-        assert np.all(F.asnumpy(partids2 == part_id)), 'load a wrong partition'
-        etypes_list.append(etype)
+    ntypes_list = list(ntypes.keys())
+    etypes_list = list(etypes.keys())
+    if 'DGL_DIST_DEBUG' in os.environ:
```

Comment:
We really need a docpage about all the environment variables used in DistDGL. A temporary solution is to add a page to the distributed training user guide.

Reply:
how about achieving this in another dedicated PR?

Reply:
sure